tests/LineBot.Tests/RichMenuRequestTests/RichMenuTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright 2017-2018 Dirk Lemstra (https://github.com/dlemstra/line-bot-sdk-dotnet)
+//
+// Dirk Lemstra licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Line.Tests
+{
+    [TestClass]
+    public class RichMenuTests
+    {
+        [TestMethod]
+        public async Task ShouldThrowExceptionWhenRichMenuIsNull()
+        {
+            ILineBot bot = TestConfiguration.CreateBot();
+            await ExceptionAssert.ThrowsArgumentNullExceptionAsync("richMenuRequest", async () =>
+            {
+                await bot.CreateRichMenu(null);
+            });
+        }
+
+        [TestMethod]
+        public async Task ShouldNotThrowExceptionWhenRichMenuIsGood()
+        {
+            var richMenu = new TestRichMenuRequest();
```

Comment:
Can we use the `RichMenu` class here instead of use `TestRichMenuRequest` inside the unit tests for the `Convert` method? I can handle that part if you would like me to.

Reply:
Great, I'm not sure about this part.